src/vs/workbench/contrib/chat/browser/chatViewPane.ts

```diff
@@ -229,14 +229,14 @@ export class ChatViewPane extends ViewPane implements IViewWelcomeDelegate {
 			}));
 		this._widget.render(parent);
 
-		const updateWidgetVisibility = () => {
```

Comment:
My recommendation would be to inline `updateWidgetVisibility`. In that case, the lint rule would warn you about the `get` inside of an autorun/derived.
Also, it seems like `isBodyVisible` is not observable? Can it update?